frontend/src/components/maps/leaflet/Map.tsx

```diff
@@ -160,16 +160,20 @@ const Map: React.FC<MapProps> = ({
   });
 
   const lastZoom = useSelector<RootState, number>(state => state.mapViewZoom) ?? zoomProp;
+  const [zoom, setZoom] = useState(lastZoom);
   useEffect(() => {
-    dispatch(setMapViewZoom(smallScreen ? 4.9 : 5.5));
-  }, [dispatch, smallScreen]);
+    if (lastZoom === DEFAULT_MAP_ZOOM) {
```

Comment:
PA-2305 actually update lastzoom when the zoom changes, previously lastzoom is only set to a default value and never changed again. This is what was causing the map to zoom out.